crd-docs/crd/kiali.io_kialis.yaml

```
@@ -1297,9 +1297,18 @@ spec:
                             description: "Specific properties when the collector type is `otel`."
                             type: object
                             properties:
+                              ca_name:
+                                description: "The name of the ca name containing when `tls_enabled` is `true` and `skip_verify` is `false`."
+                                type: string
                               protocol:
                                 description: "Protocol. Supported values are: `http`, `https` or `grpc`."
                                 type: string
+                              skip_verify:
+                                description: "Skip tls verification when using self signed certificates. Insecure option recommended just for testing."
```

This doesn't just skip verification only if using self-signed certs. If you set this to true, NO cert verification will be done, regardless of where the CA came from. We should change this to make that more clear.
Also, capitalize TLS.

```suggestion
                                description: "If true, TLS certificate verification will not be performed. This is an unsecure option and is recommended only for testing."
```